tests/mail/tests.py

```diff
@@ -60,6 +60,27 @@ class MailTests(HeadersCheckMixin, SimpleTestCase):
     """
     Non-backend specific tests.
     """
+    def get_decoded_attachments(self, django_message):
+        """
+        Encodes the specified django.core.mail.message.EmailMessage, decodes
+        it using pythons email.parser module and for each attachment of the
+        message, returns a tuple with (filename, content, mimetype).
+        """
+        msg_bytes = django_message.message().as_bytes()
+        email_message = message_from_bytes(msg_bytes)
+
+        def iter_attachments():
+            for i in email_message.walk():
+                # get_content_disposition() was added in Python 3.5.
```

Comment:
I guess this means that the line below could be replaced when only Python 3.5+ is supported. It would be nice to be more explicit about that.